### DISCUSSION TITLE
gained pp on osu!lazer?

### DISCUSSION THREAD

**Discussion by xeroine** [ppy/osu#20942] [Q&A] [answered]
I recently created a new account for osu! and have only played osu!lazer so far since it has an AppImage for Linux (will try osu! stable on wine after making this post).
I've played for around 3 days and noticed I currently have 3 pp. I thought it's impossible to gain pp from playing on osu!lazer, has this changed recently in the last few days?
![osu1](https://user-images.githubusercontent.com/74143110/197940671-5e5a68a4-ab67-4da8-8b0f-2f1401c40545.png)
![osu2](https://user-images.githubusercontent.com/74143110/197941029-c6b1d8f8-b7b4-437d-a7c1-2cbff3b941e4.png)

**Comment by xeroine** [comment]
I'm new to osu! so I'm not sure if I'm misunderstanding something about how pp works but the 3 pp I gained on osu!lazer took me a long time to get, I played for several days on osu!lazer on more than 10 maps with different star ratings and difficulties but gained only 3 pp from those 2 maps. Now I tried out osu! stable and it looks like I'm gaining it after every game. Just played 2 games on osu! stable and this is what it looks like now
![osu3](https://user-images.githubusercontent.com/74143110/197944499-ac61725d-ae60-48f2-a2b0-f4444198763d.png)

**Comment by peppy** [comment, marked as answer]
Right now, the "total pp" number is not yet updated from scores set on osu!lazer. Individual scores will have the correct values for both version of the game. This will change in the near future.

**Reply by xeroine** [reply]
But like I said I didn't even open osu! stable version util today. Unless I misunderstood you, there should be no way to gain pp by playing on osu!lazer yet it looks like I somehow gained 3 pp while only playing on osu!lazer.

**Reply by peppy** [reply]
Lazer scores do not show up on "best performance", so you definitely somehow played non-lazer.

**Reply by xeroine** [reply]
Jesus christ you're right, I'm sorry my memory is just so bad recently. I was sure I didn't.
